### ISSUE TITLE
[1remote] Package is installed OK, but whenever launched automatically on OS startup it is elevated

### ISSUE THREAD

**Issue by b-a0** [majkinetor/au-packages#251]
See the description in this 1Remote issue: https://github.com/1Remote/1Remote/issues/811

**Comment by majkinetor** [comment]
I did nothing to elevate.

Probably due to this:

> First, ensure that you are using an [administrative shell](https://www.howtogeek.com/194041/how-to-open-the-command-prompt-as-administrator-in-windows-10/) - you can also install as a non-admin, check out [Non-Administrative Installation](https://docs.chocolatey.org/en-us/choco/setup#non-administrative-install).

I don't think there is anything I can do as this seems to be user installation specific. Have you tried to install it in non-admin shell and ignore the warning?

**Comment by b-a0** [comment]
No didn't try that, as I manage almost all software on this machine using chocolatey and it's hard to track what software should be installed with the admin/non-admin shell. Other software that autostarts and is installed with chocolatey does not seem to have this problem (e.g. [greenshot](https://community.chocolatey.org/packages/greenshot)) so something in the 1Remote autostart mechanism seems to interact with chocolatey in an unexpected way.

**Comment by majkinetor** [comment]
Nah, package doesn't do anything specific, it doesn't even add to startup.

https://github.com/majkinetor/au-packages/blob/37606b36ac3f62bea738ebce53b232a96c35fe32/1remote/tools/chocolateyInstall.ps1

Here is my registry entry:

![image](https://github.com/user-attachments/assets/835fbca9-1b43-4fe0-863b-57e94f1e10c6)

It definitelly is not about the package.

**Comment by majkinetor** [comment]
So, the package only unpacks the app and adds shortcut. Everything else happens on the first run.
